vpd-tool/include/constants.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <cstdint>
+
+namespace vpd
+{
+namespace constants
+{
+static constexpr auto KEYWORD_SIZE = 2;
```

Comment:
nit: Can we use `{}` initialization?

Reply:
Kept this way, as we are following this style in constants declaration.
Unless any other way does make significant difference, its fine to declare this way .